src/genetic_forensic_portal/app/pages/5_Get_Analysis_Status.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import annotations
+
+import streamlit as st
+
+from genetic_forensic_portal.app.client import gf_api_client as client
+
+st.title("Analysis Status Checker")
```

Comment:
For now, let's have this be consistent with the filename/URL of the page

```suggestion
st.title("Get Analysis Status")
```